src/app/components/media/Similarity/MediaSimilarityBar.js

```diff
@@ -21,7 +21,6 @@ const MediaSimilarityBar = ({ projectMedia, setShowTab }) => {
             permissions
             report_status
             hasMain: is_confirmed_similar_to_another_item
-            isSuggested: is_suggested
```

Comment:
I snuck it in with this commit actually @dariusk 
I did the same thing you did and it ran fine